resources/sass/components/_filters.scss

```diff
@@ -18,31 +27,46 @@
     }
 
     &__subheader {
-        padding: $indent-m;
+        color: $c-gray2;
+        display: flex;
+        padding: $indent-sm;
+
+        &:hover {
+            cursor: pointer;
+        }
 
         &_has-icon {
             &::after {
+                align-self: center;
```

Comment:
better to use separate element for icons, for example `<span class="..."></span>` and not redefine icon on the whole component with `:after` or `:before`